### ISSUE TITLE
Support soundcloud.

### ISSUE THREAD

**Issue by heliomeiralins** [vintasoftware/hub.rocks#42]

**Comment by fjsj** [comment]
maybe this was closed by mistake?

**Comment by heliomeiralins** [comment]
not really. I just think it doesn't make sense to add soundcloud support anymore.

maybe we could accept soundcloud direct links and not search them, but Im not sure if its worth the effort. =)
